x-pack/plugins/security_solution/server/lib/detection_engine/rules/get_export_rule_exceptions.ts

```diff
@@ -21,10 +22,18 @@ export const getRuleExceptionsForExport = async (
   exceptions: ListArray,
   exceptionsListClient: ExceptionListClient | undefined
 ): Promise<ExportExceptionListAndItemsReturn> => {
+  const uniqueExceptionLists = new Set();
+
   if (exceptionsListClient != null) {
-    const exceptionsWithoutUnexportableLists = exceptions.filter(
-      ({ list_id: listId }) => !NON_EXPORTABLE_LIST_IDS.includes(listId)
-    );
+    const exceptionsWithoutUnexportableLists = exceptions.filter((list) => {
+      const listHash = createHash('sha256').update(JSON.stringify(list)).digest('hex');
+      if (!uniqueExceptionLists.has(listHash)) {
+        uniqueExceptionLists.add(listHash);
+        return !NON_EXPORTABLE_LIST_IDS.includes(list.list_id);
```

Comment:
Why are you using a hash here and not just a unique identifier such as the `list_id`? Shouldn't that be enough for this to work. If two are the same list but they are different in ordering then the hash will begin to fail on you.

Reply:
I had gone with this because I worried about possibilities of the `list_id` being the same once they become shareable. Although, at that point I think something else would be wrong. I'll update to just use the `id` as those should be globally unique.